galsim/table.py

```diff
@@ -564,7 +564,7 @@ def __eq__(self, other):
                 self.edge_mode == other.edge_mode)
 
     def __ne__(self, other):
-        return not self.__eq__(self, other)
+        return not self.__eq__(other)
```

Comment:
Didn't used to.  But we do now.  I added tests of the `!=` operator in the `do_pickle` tests that we run for everything.